composer.json

```diff
@@ -12,7 +12,10 @@
     ],
     "require": {
         "php": ">=5.5",
-        "dragonbe/vies": "^1.0"
+        "dragonbe/vies": "1.0.6",
+        "symfony/http-kernel": "^2.8",
```

Comment:
Why have these symfony packages been added as requirement? Are they not dependencies for `dragonbe/vies` 1.0.6?